status/raw/status-app-protocols.md

```diff
@@ -0,0 +1,372 @@
+1. Functional scope
+2. Content topic
+3. Ephemerality
+
+### Functional Scope
+
+Each Status app-level message MUST define a functional scope.
+The functional scope MUST define the _minimum_ scope of the audience that should _participate_ in the app function the message is related to.
+In other words, it determines the minimum subset of Status app participants
+that should have access to messages related to that function.
+
+Note that the functional scope is distinct from the number of participants that is _addressed_ by a specific message.
+For example, a participant will address a 1:1 chat to only one other participant.
+However, since all users of the Status app MUST be able to participate in 1:1 chats,
+the functional scope of messages enabling 1:1 chats MUST be a global scope.
+Similarly, since private group chats can be set up between any subset of Status app users,
+the functional scope for messages related to private group chats MUST be global.
+As a counter-example, messages that originate within a community (and are _addressed_ to members of that community)
+are only of interest to participants that are also members of that community.
+Such messages MUST have a community-wide functional scope.
+A third group of messages are addressed only to the participant that generated those messages itself.
+These _self-addressed_ messages MUST have a local functional scope.
+
+If we further make a distinction between "control" and "content" messages,
+we can distinguish five distinct functional scopes.
+
+All Status messages MUST have one of these functional scopes:
+
+**Global scope**
+
+1. _Global control_: messages enabling the basic functioning of the app to control features that all app users should be able to participate in. Examples include Contact Requests, Community Invites, global Status Updates, Group Chat Invites, etc.
+2. _Global content_: messages carrying user-generated content for global functions. Examples include 1:1 chat messages, images shared over private group chats, etc.
+
+**Community scope**
+
+3. _Community control_: messages enabling the basic functioning of the app to control features _only relevant to members of a specific community_. Examples include Community Membership Updates, community Status Updates, etc.
+4. _Community content_: messages carrying user-generated content _only for members of a specific community_.
+
+**Local scope**
+
+5. _Local_: messages related to functions that are only relevant to a single user. Also known as _self-addressed messages_. Examples include messages used to exchange information between app installations, such as User Backup and Sync messages.
+
+Note that the functional scope is a logical property of Status messages.
+It SHOULD however inform the underlying [transport layer sharding](#pubsub-topics-and-sharding) and [transport layer subscriptions](#subscribing) underneath the hood.
```

Comment:
nitpick: _underlying_ and _underneath_ are redundant

Reply:
Thanks. Fixed in [9ce3bc5](https://github.com/vacp2p/rfc-index/pull/105/commits/9ce3bc5700324e9172729808b4af9eb68d9a9a91)